test/e2e/catalog_e2e_test.go

```diff
@@ -272,6 +269,20 @@ var _ = Describe("Catalog represents a store of bundles which OLM can use to ins
 		})
 		Expect(err).ShouldNot(HaveOccurred())
 
+		var updatedConfigMap *corev1.ConfigMap
+		Eventually(func() (types.UID, error) {
+			var err error
+			// Get updated configmap
+			updatedConfigMap, err = c.KubernetesInterface().CoreV1().ConfigMaps(testNamespace).Get(context.Background(), fetchedInitialCatalog.Spec.ConfigMap, metav1.GetOptions{})
+			if err != nil {
+				return "", err
+			}
+			if len(updatedConfigMap.ObjectMeta.OwnerReferences) == 0 {
+				return "", nil
+			}
+			return updatedConfigMap.ObjectMeta.OwnerReferences[0].UID, nil
+		}).Should(Equal(fetchedUpdatedCatalog.ObjectMeta.UID))
```

Comment:
What happens when an error is returned here? It looks like we only account for the UID comparison if I'm reading this correctly. Should we just return an empty string when an error is encountered, or just extend the Eventually to also check whether the error is nil?

Reply:
Actually I don't think I'm super concerned with the error path here anymore, and I'm seeing this flake pop up more frequently recently, so I'm down to just push this through.

/lgtm